.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -55,19 +76,36 @@ jobs:
       uses: "./.github/actions/step_create_python_environment"
       with:
         pythonVersion: ${{ steps.display_env.outputs.pyVersion }}
-    - name: Build wheel
-      uses: "./.github/actions/step_sdk_setup"
+    - name: Download Artifacts
```

Comment:
Nit: can we run build & test in same vm to make it faster?

Reply:
Separate build stage to prevent there are some unnecessary packages included inside testing environment.